portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java

```diff
@@ -56,11 +60,31 @@ public void init() {
 
       Double riceConsump = riceStr.equals(UNDEFINED_DATA) ? 0.0 : Double.valueOf(riceStr);
 
-      countryMap.put(country, riceConsump);
+      if (countryNameMap.containsKey(country)) {
+        countryMap.put(countryNameMap.get(country), riceConsump);
+      } else {
+        countryMap.put(country, riceConsump);
+      }
```

Comment:
This can also be consolidated: 

```suggestion
      String countryName = countryNameMap.containsKey(country) ? countryNameMap.get(country) : country;
      countryMap.put(countryName, riceConsump);
```